src/main/java/project/dao/jdbc/CarDaoJdbc.java

```diff
@@ -0,0 +1,225 @@
+package project.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import project.dao.CarDao;
+import project.exception.DataProcessingException;
+import project.lib.Dao;
+import project.models.Car;
+import project.models.Driver;
+import project.models.Manufacturer;
+import project.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbc implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String selectQuery = "INSERT INTO cars (car_model, manufacturer_id) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement stInsert = connection.prepareStatement(selectQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            stInsert.setString(1, car.getName());
+            stInsert.setLong(2, car.getManufacturer().getId());
+            stInsert.execute();
+            ResultSet resultSet = stInsert.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject("GENERATED_KEY", Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car " + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectQuery = "SELECT c.car_id, c.car_model,"
+                + "c.manufacturer_id, cd.driver_id FROM cars c "
+                + "INNER JOIN manufacturers m ON m.manufacturer_id=c.manufacturer_id "
+                + "INNER JOIN cars_drivers cd ON cd.car_id=c.id "
+                + "INNER JOIN drivers d ON d.driver_id=cd.driver_id "
+                + "WHERE c.deleted=FALSE AND d.deleted=FALSE AND c.car_id=?";
```

Comment:
```suggestion
                + "INNER JOIN manufacturers m ON m.manufacturer_id=c.manufacturer_id "
                + "WHERE c.deleted=FALSE AND c.car_id=?";
```
we don't need to fetch info about drivers here